deploy/cloud-foundry/Jenkinsfile

```diff
@@ -54,6 +57,17 @@ pipeline {
         }
       }
     }
+    stage('Register with ELK') {
+      agent any // Don't run in docker container.
+      when {
+        expression { return env.ELK_REGISTRATION_COMMAND }
```

Comment:
I am not sure this will work as intended.

It may need to be:
```groovy
when {
  not {
    environment name: 'ELK_REGISTRATION_COMMAND', value: ''
  }
}
```